### DISCUSSION TITLE
ERR_ABORTED 404 error

### DISCUSSION THREAD

**Discussion by SingingFoot** [community/community#23616] [Other Feature Feedback, Questions, & Ideas]
<p><a href="https://singingfoot.github.io/ptakh/" rel="noopener nofollow ugc">My site is running well</a> on Live Server while opened in VSC, but when I published it on GitHub Pages it runs without CSS files. When I opened an inspector I found this ERR_ABORTED 404 error in Console:</p>
<p><img src="https://user-images.githubusercontent.com/68031723/181078244-3ecd0b2c-d53d-4b12-8de6-c61c9aa1c45c.png" alt="Screenshot 2021-12-29 at 21.02.31" data-base62-sha1="8fPLkKEoyMxxWfZfPbiRHDucQKm" width="564" height="112"></p>
<p>This is the reason why my CSS file does not work on GitHub Pages. But it works well on Live Server. What should I do?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>You’re using an absolute path for the CSS file:</p><aside class="onebox githubblob" data-onebox-src="https://github.com/SingingFoot/ptakh/blob/04599b4361230fe5aa6812072694ff7e1eddf83b/index.html#L7">
  <header class="source">

      <a href="https://github.com/SingingFoot/ptakh/blob/04599b4361230fe5aa6812072694ff7e1eddf83b/index.html#L7" target="_blank" rel="noopener">github.com</a>
  </header>

  <article class="onebox-body">
    <h4><a href="https://github.com/SingingFoot/ptakh/blob/04599b4361230fe5aa6812072694ff7e1eddf83b/index.html#L7" target="_blank" rel="noopener">SingingFoot/ptakh/blob/04599b4361230fe5aa6812072694ff7e1eddf83b/index.html#L7</a></h4>



  <pre class="onebox">    <code class="lang-html">
      <ol class="start lines" start="1" style="counter-reset: li-counter 0 ;">
          <li>&lt;!DOCTYPE html&gt;</li>
          <li>&lt;html&gt;</li>
          <li>   &lt;head&gt;</li>
          <li>      &lt;meta charset="UTF-8"&gt;</li>
          <li>      &lt;meta http-equiv="X-UA-Compatible" content="IE=edge"&gt;</li>
          <li>      &lt;meta name="viewport" content="width=device-width, initial-scale=1.0"&gt;</li>
          <li class="selected">      &lt;link rel="stylesheet" href="/css/style.css"&gt;</li>
          <li>      &lt;title&gt;Згодуй птахові ногу свою!&lt;/title&gt;</li>
          <li>   &lt;/head&gt;</li>
          <li>   &lt;body&gt;</li>
          <li>      &lt;div class="container"&gt;</li>
          <li>         &lt;h1 class="title"&gt;Зміст&lt;/h1&gt; </li>
          <li>      &lt;ul&gt;</li>
          <li>         &lt;li&gt;&lt;p&gt;&lt;a href="1.html" class="href"&gt;Частина перша&lt;/a&gt;&lt;/p&gt;&lt;/li&gt;</li>
          <li>         &lt;li&gt;&lt;p&gt;&lt;a href="2.html" class="href"&gt;Частина друга&lt;/a&gt;&lt;/li&gt;</li>
          <li>         &lt;li&gt;&lt;p&gt;&lt;a href="3.html" class="href"&gt;Частина третя&lt;/a&gt;&lt;/p&gt;&lt;/li&gt;</li>
          <li>         &lt;li&gt;&lt;p&gt;&lt;a href="4.html" class="href"&gt;Частина четверта&lt;/a&gt;&lt;/p&gt;&lt;/li&gt;</li>
      </ol>
    </code>
  </pre>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>That breaks because your page isn’t at the root of the server. Change it to a relative URL (without the leading <code>/</code>) so it works as long as the relative position of your HTML and CSS files is the same.</p>

**Comment by SingingFoot** [comment]
<p>Thanks! That helped!</p>
